aclk/aclk.c

```diff
@@ -992,6 +993,8 @@ void ng_aclk_host_state_update(RRDHOST *host, int cmd)
     uuid_unparse_lower(node_id, (char*)query->data.node_update.node_id);
     query->data.node_update.queryable = 1;
     query->data.node_update.session_id = aclk_session_newarch;
+    info("Sending status update for node=%s, live=%d, hops=%u",(char*)query->data.node_update.node_id, cmd,
```

Comment:
i worry about this being misleading in future as we queue here not send?

Reply:
I can remove those (keep it in the "testing" branch) or rename to "Queuing" status update.

Reply:
Adjusted to use "Queuing"